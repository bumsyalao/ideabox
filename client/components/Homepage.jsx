import React from 'react';
import { Link } from 'react-router-dom';
import ideaboxlogo from '../../client/images/idea logo.png';

/**
 * @function Homepage
 * @return {void}
 */
const Homepage = () => (
  <div>
    { $('.tooltipped').tooltip('remove') }
    <div className="section no-pad-bot bkg-style" id="index-banner">
      <div className="homepage">
        <br />
        <div className="center">
          <img src={ideaboxlogo} />
        </div>
        <div className="row center">
          <h3 className="header col s12 light">Think inside the box!</h3>
        </div>
        <div className="row center">
          <Link
            to="/auth"
            id="download-button"
            className="btn-large waves-effect waves-light amber black-text"
          >
            Get Started
          </Link>{' '}
        </div>
        <br />
      </div>
    </div>
  </div>
);

export default Homepage;
