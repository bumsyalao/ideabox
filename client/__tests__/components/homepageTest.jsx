/* global expect test */
import React from 'react';
import { shallow, configure } from 'enzyme';
import toJson from 'enzyme-to-json';
import adapter from 'enzyme-adapter-react-16';
import Homepage from '../../components/Homepage';

configure({ adapter: new adapter() });

describe('Hompeage component', () => {
  const component = shallow(
    <Homepage />
  );
  it('should match the homepage component snapshot', () => {
    const tree = toJson(component);
    expect(tree).toMatchSnapshot();
  });
});
