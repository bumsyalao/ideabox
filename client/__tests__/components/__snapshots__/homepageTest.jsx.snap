// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Hompeage component should match the homepage component snapshot 1`] = `
<div>
  <div
    className="section no-pad-bot bkg-style"
    id="index-banner"
  >
    <div
      className="homepage"
    >
      <br />
      <div
        className="center"
      >
        <img
          src={
            Object {
              "process": [Function],
            }
          }
        />
      </div>
      <div
        className="row center"
      >
        <h3
          className="header col s12 light"
        >
          Think inside the box!
        </h3>
      </div>
      <div
        className="row center"
      >
        <Link
          className="btn-large waves-effect waves-light amber black-text"
          id="download-button"
          replace={false}
          to="/auth"
        >
          Get Started
        </Link>
         
      </div>
      <br />
    </div>
  </div>
</div>
`;
